ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PutBulk.java

```diff
@@ -69,9 +74,18 @@ public CliCommand init(final Arguments args) throws Exception {
             throw new MissingOptionException("The bulk put command requires '-b' to be set.");
         }
 
-        final String srcDir = args.getDirectory();
-        if (srcDir == null) {
-            throw new MissingOptionException("The bulk put command required '-d' to be set.");
+        pipe = args.isPipe();
+        if (pipe) {
+            this.pipedFiles = getPipedFilesFromStdin();
```

Comment:
What kind of validation are we performing against the files that we are reading from stdin?